data/mods/gen3/abilities.js

```diff
@@ -63,23 +63,15 @@ let BattleAbilities = {
 	"intimidate": {
 		inherit: true,
 		onStart(pokemon) {
-			let activated = false;
-			for (const target of pokemon.side.foe.active) {
-				if (target && this.isAdjacent(target, pokemon) && !target.volatiles['substitute']) {
-					activated = true;
-					break;
-				}
-			}
+			let activated = pokemon.foes().some(foe => !foe.volatiles['substitute']);
```

Comment:
should this and below not be `adjacentFoes`?

Reply:
There's no triples in gen 3, so no need to check for adjacency.